sdk/communication/communication-phone-numbers/test/public/siprouting/deleteTrunk.spec.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import { Context } from "mocha";
+
+import { SipRoutingClient } from "../../../src";
+
+import { Recorder } from "@azure-tools/test-recorder";
```

Comment:
You should upgrade to @azure-tools/test-recorder^2.0.0 and use the new recorder instead.

Here is more information on how to migrate: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/MIGRATION.md

Reply:
Upgraded to @azure-tools/test-recorder^2.0.0. Please verify.